src/passes/I64ToI32Lowering.cpp

```diff
@@ -1526,7 +1520,28 @@ struct I64ToI32Lowering : public WalkerPass<PostWalker<I64ToI32Lowering>> {
   }
 
   void visitSelect(Select* curr) {
-    visitBranching<Select>(curr);
+    if (!hasOutParam(curr->ifTrue)) return;
+    TempVar trueBits = fetchOutParam(curr->ifTrue);
+    TempVar highBits = fetchOutParam(curr->ifFalse);
```

Comment:
perhaps `falseBits` to match the line above?

Reply:
done